client/me/profile/index.jsx

```diff
@@ -100,7 +100,7 @@ const Profile = createReactClass( {
 
 						<FormFieldset>
 							<FormLabel htmlFor="display_name">
-								{ this.props.translate( 'Public Display Name' ) }
+								{ this.props.translate( 'Public display name' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.